web/client/components/style/vector/Text.jsx

```diff
@@ -194,8 +195,37 @@ class Text extends React.Component {
                     />
                 </Col>
             </Row>
+            <Row>
+                <Col xs={6}>
+                    <Message msgId="draw.textRotation"/>
+                </Col>
+                <Col xs={6} style={{position: "static"}}>
+                    <div className="mapstore-slider with-tooltip">
+                        <Slider
+                            tooltips
+                            step={2}
```

Comment:
With step 2 you cannot set 45°, and i don't think we need more granular setup,
I think is a good idea to make this parameter configurable from configuration with a default of 5 if not configured.
so please provide this possibility
```suggestion
                            step={this.props.rotationStep}
```